tools/Makefile

```diff
@@ -39,13 +63,21 @@ spiffsimg: spiffsimg/spiffsimg
 spiffsimg/spiffsimg: 
 	@$(MAKE) -C spiffsimg CC=$(HOSTCC)
 
-spiffsscript: remove-image spiffsimg/spiffsimg
+spiffsscript: remove-image LFSimage spiffsimg/spiffsimg
 	rm -f ./spiffsimg/spiffs.lst
-	echo "" >> ./spiffsimg/spiffs.lst
+	@echo "" >> ./spiffsimg/spiffs.lst
 	@$(foreach f, $(SPIFFSFILES), echo "import $(FSSOURCE)$(f) $(f)" >> ./spiffsimg/spiffs.lst ;)
-	@$(foreach sz, $(FLASHSIZE), spiffsimg/spiffsimg -U $(FLASH_USED_END) -o ../bin/spiffs-$(sz).dat -f ../bin/0x%x-$(sz).bin -S $(sz) -r ./spiffsimg/spiffs.lst -d; )
+	$(foreach sz, $(FLASHSIZE), spiffsimg/spiffsimg -f ../bin/0x%x-$(sz).img  $(FLASH_SW) $(sz) -U $(FLASH_FS_LOC) -r ./spiffsimg/spiffs.lst -d; )
 	@$(foreach sz, $(FLASHSIZE), if [ -r ../bin/spiffs-$(sz).dat ]; then echo Built $$(cat ../bin/spiffs-$(sz).dat)-$(sz).bin; fi; )
-	
+
+ifneq ($(LFSSOURCES),)
+LFSimage: $(LFSSOURCES)
+       ../luac.cross -f -o $(FSSOURCE)flash.img $(LFSSOURCES)
```

Comment:
Spaces instead of a TAB. Causes `Makefile:75: *** missing separator.  Stop.`

Reply:
Bloody `gedit` preference.  Already picked this on up.